README.md

```diff
@@ -241,3 +241,123 @@ docker run                         \
 curl http://localhost:8001/
 ```
 
+### Virtual Machine that allow create docker components for debugging purpose
+
+Step-by-step instruction to building and running virtual machine with all needed dockers.
+
+- Install require libs:
```

Comment:
please add which libs or/and apps are required before installation and in which version (if there could be problems with compatibility).

Reply:
@pawelkisielewicz This is not the right issue. This code was actually added in #159. Please add your remarks there. I'm changing the base branch here so that the pull request only shows the code that was added.